wysadzulice/assets/js/new_campaign.js

```diff
@@ -0,0 +1,96 @@
+import jquery from 'jquery';
+import Planting from 'plantingjs';
+import { Model, View } from 'backbone';
+
+
+const NewCampaignMapsView = View.extend({
+
+  initialize: function(options) {
+    this.parent = options.parent;
+    this.engine = new Planting({
+      container: this.$el,
+      manifestoUrl: options.manifestoUrl,
+      googleApiKey: options.googleApiKey,
+      saveUrl: this.parent.saveUrl,
+      selectPanoMode: true,
+      onSelectPano: this.parent.on_select_pano.bind(this.parent),
+    });
+  },
+
+});
+
+
+const ItemView = View.extend({
+
+  initialize: function(options) {
+    this.$el.html('<img src="' + options.image + '"></img>');
```

Comment:
Regarding tagged strings - You're right, I forgot about them.

Regarding separate view - I introduce it, because I had in mind that there will be another story about this items and I believe we will use some events in this separate view.